itou/utils/apis/api_particulier.py

```diff
@@ -11,94 +11,94 @@
 logger = logging.getLogger("APIParticulierClient")
 
 
-class APIParticulierClient:
-    def __init__(self, job_seeker=None):
-        self.client = httpx.Client(
-            headers={"X-Api-Key": settings.API_PARTICULIER_TOKEN}, base_url=settings.API_PARTICULIER_BASE_URL
-        )
-        self.job_seeker = job_seeker
+def client():
+    return httpx.Client(
+        base_url=settings.API_PARTICULIER_BASE_URL,
+        headers={"X-Api-Key": settings.API_PARTICULIER_TOKEN},
+    )
 
-    @staticmethod
-    def format_date(date: str) -> datetime.datetime:
-        return datetime.datetime.strptime(date, "%Y-%m-%d") if date else ""
 
-    @classmethod
-    def _build_params_from(cls, job_seeker):
-        jobseeker_profile = job_seeker.jobseeker_profile
-        requested_objects = [
-            jobseeker_profile.birth_country,
-            jobseeker_profile.birthdate,
-            job_seeker.first_name,
-            job_seeker.last_name,
-            job_seeker.title,
-        ]
-        # TODO(cms): add JobSeekerProfile.is_born_in_france
-        born_in_france = (
-            jobseeker_profile.birth_country and jobseeker_profile.birth_country.group == Country.Group.FRANCE
-        )
-        if born_in_france:
-            requested_objects.append(jobseeker_profile.birth_place)
-        if not all(requested_objects):
-            raise KeyError(f"Missing parameters for {job_seeker.public_id=}. Unable to call the API Particulier.")
+def _parse_date(date: str) -> datetime.date | None:
+    if date:
+        return datetime.datetime.fromisoformat(date)
```

Comment:
Le type hint n'est pas bon

Après vu que ça finit dans un `DateRangeField`, ce serait effectivement plus pertinent de retourner une date :

```suggestion
def _parse_date(date: str) -> datetime.date | None:
    if date:
        return datetime.date.fromisoformat(date)
```